server.py

```diff
@@ -0,0 +1,234 @@
+import requests
+from flask import Flask, request, send_file, jsonify
+import io
+from flask_cors import CORS
+from timetable import build_courses, generate_ics
+import logging
+import iitkgp_erp_login.erp as erp
+import iitkgp_erp_login.utils as erp_utils
+from typing import Dict, List
+from utils.dates import *
+
+
+app = Flask(__name__)
+CORS(app)
+
+headers = {
+    "timeout": "20",
+    "User-Agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.3"
+}
+
+def check_missing_fields(all_fields: Dict[str, str]) -> List[str]:
+    return [field for field, value in all_fields.items() if not value]
+
+class ErpResponse:
+    def __init__(
+        self,
+        success: bool,
```

Comment:
iirc, success was set to True by default and when false it was specified separately. Make changes accordingly.